src/modules/esl-mixin-element/test/mixin.test.ts

```diff
@@ -101,4 +101,108 @@ describe('ESLMixinElement', () => {
       while (document.body.lastElementChild) document.body.removeChild(document.body.lastElementChild);
     });
   });
+
+  describe('ESLMixinElement prototype', () => {
+    class CTestMixin extends ESLMixinElement {
+      static override is = 'c-test';
+    }
+
+    const $host = document.createElement('div');
+    $host.toggleAttribute(CTestMixin.is, true);
+    CTestMixin.register();
+
+    beforeAll(() => document.body.appendChild($host));
+
+    describe('ESLMixinElement $attr', () => {
+      let $el: CTestMixin;
+      const attrName = 'test-attr';
+
+      beforeAll(() => $el = CTestMixin.get($host) as CTestMixin);
```

Comment:
Consider moving and merging with beforeAll on the top level to get rid of repetitive code
To deal with async nature of registration.
Register mixin after adding it to the dom or make beforeAll step async with a microtask waiter trick:
`await Promise.resolve()`